### ISSUE TITLE
Cannot tab/focus on period picker control

### ISSUE THREAD

**Issue by AdamWillden** [xdan/periodpicker#48]
For accessibility reasons I think the user should be able to tab onto the control and open it. But the generated control is a div and not a button.

Below is a snippet from your code which shows `type="button"`. I don't know if changing this would fix it or if it would have some unanticipated side effects:

``` js
that.button = $('<div class="period_picker_input" type="button">' +
```

**Comment by xdan** [comment]
Added tabIndex option
Try here http://xdsoft.net/jqplugins/periodpicker/
